packages/mui-joy/src/List/List.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import { OverridableComponent } from '@mui/types';
+import composeClasses from '@mui/base/composeClasses';
+import { styled, useThemeProps } from '../styles';
+import { ListProps, ListTypeMap } from './ListProps';
+import { getListUtilityClass } from './listClasses';
+
+const useUtilityClasses = () => {
+  const slots = {
+    root: ['root'],
+  };
+
+  return composeClasses(slots, getListUtilityClass, {});
+};
+
+const ListRoot = styled('ul', {
+  name: 'MuiList',
+  slot: 'Root',
+  overridesResolver: (props, styles) => styles.root,
+})<{ ownerState: ListProps }>({
+  '--List-padding': '0.375rem',
+  '--List-radius': '8px',
+  '--List-itemMinHeight': '2.5rem',
+  '--List-itemGutter': '0.375rem',
```

Comment:
```suggestion
  '--List-itemHorizontalPadding': '0.375rem',
```
As far as I understood, `List-itemGutter` only affects padding on the X-axis, is that right? I feel like _gutter_ is a confusing word, not too commonly used anymore (I could be wrong!). I know the suggestion gets more verbose-ish, but maybe it's clearer?

Reply:
> List-itemGutter only affects padding on the X-axis

Correct.

> I know the suggestion gets more verbose-ish, but maybe it's clearer?

Absolutely, I agree. In my opinion, the ideal name should reflect what you would get if you change the value.

Reply:
What's about `--List-itemPaddingX`?

Reply:
I thought of it first but then thought that it could be initially confusing what X is for, but I guess we could go with it, to match with `px` from the sx prop.